tests/webapp/api/test_bug_job_map_api.py

```diff
@@ -148,3 +162,17 @@ def test_bug_job_map_delete(webapp, eleven_jobs_stored, test_repository,
         content = json.loads(resp.content)
         assert content == {"message": "Bug job map deleted"}
         assert BugJobMap.objects.count() == 0
+
+
+def test_bug_job_bad_job_id(webapp, test_repository):
+    """
+    test we have graceful error when we pass an invalid job_id
+    """
+    bad_job_id = "aaaa"
+
+    resp = webapp.get(
+            reverse("bug-job-map-list", kwargs={"project": test_repository.name}),
```

Comment:
The PR currently fixes one issue (in the `list` view for the bug-job-map API), but adds two tests, when I would have expected just one. 

The extra test (`test_bug_job_map_list_bad_job_id`) has a name that implies it's trying to test the `detail` view for the bug-job-map API, but if that's the case:
(a) the contents of the test aren't correct, since it's actually duplicating `test_bug_job_bad_job_id`, when it would need to be using `reverse("bug-job-map-detail", ..."` instead.
(b) then the PR would also need to be updated, since the detail view is actually broken currently (ie needs the try-except too).

So therefore the options are to either:
(a) remove the duplicate test and open a new bug/PR
(b) fix the test and the detail view in the same PR

I initially proposed (b), but after you mentioned wanting to do this in another PR was saying it's fine to do (a).